Cargo.lock

```diff
@@ -4542,6 +4566,16 @@ version = "1.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4950d85bc52415f8432144c97c4791bd0c4f7954de32a7270ee9cccd3c22b12b"
 
+[[package]]
+name = "saffron"
+version = "0.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "03fb9a628596fc7590eb7edbf7b0613287be78df107f5f97b118aad59fb2eea9"
+dependencies = [
+ "chrono",
+ "nom 5.1.3",
+]
```

Comment:
Is `saffron` maintained anymore? https://github.com/cloudflare/saffron/issues/20

Anyways, we should ask `saffron` to upgrade to `nom` 7.1

Reply:
I got confirmation that Cloudflare is happy to merge PRs, as it is still in use at CF.

Reply:
@littledivy can we do this after this PR is landed?

Reply:
Sure

Reply:
Working on an alternative that may be of interest - with second granularity and support for timezoned DateTimes. Early stuff, and (disclaimer) my first Rust project ever, but starting to look like something - https://crates.io/crates/croner/0.0.3 https://github.com/hexagon/croner-rust

Reply:
@igorzi ready to run (or to be roasted 😉):

https://github.com/Hexagon/croner-rust/releases/tag/v1.0.0